.github/workflows/eints-to-git.yml

```diff
@@ -0,0 +1,58 @@
+name: Eints to Git
+
+# This script has to be in master, despite the code being executed in the
+# "openttd-github" branch. This is because GitHub only allows "schedule" and
+# "workflow_dispatch" to work from the default branch.
+
+on:
+  schedule:
+  - cron: '0 19 * * *'
```

Comment:
I actually do not know when the nightly runs, and whether it now runs at a fixed UTC time or still changes between CET and CETS.
However, I would expect minute 45 here.

Reply:
Workflow schedule runs UTC. Which is different from now, as indeed it currently runs on CE(S)T. I don't see a real solution atm to fix this; guess this will have to do?